client/signup/steps/domains/domains-mini-cart.jsx

```diff
@@ -78,7 +78,8 @@ export class DomainsMiniCart extends Component {
 			args: { cost: formattedOriginalCost },
 		} );
 		const hasPromotion = domain.cost_overrides?.some(
-			( override ) => ! override.does_override_original_cost
+			( override ) =>
+				! override.does_override_original_cost && override.override_code !== 'introductory-offer'
```

Comment:
I think we could add a comment here too to explain why introductory offers are not considered a discount for domains

Reply:
Done. Added a comment for clarity. Thanks!